src/core/internal/backtrace/dwarf.d

```diff
@@ -175,17 +175,28 @@ static if (hasExecinfo)
         // find address -> file, line mapping using dwarf debug_line
         Array!Location locations;
         locations.length = callstack.length;
+        size_t startIdx;
         foreach (size_t i; 0 .. callstack.length)
         {
             locations[i].address = callstack[i];
             locations[i].procedure = getMangledSymbolName(frameList[i][0 .. strlen(frameList[i])]);
+
+            // NOTE: The first few frames with the current implementation are
+            //       inside core.runtime and the object code, so eliminate
+            //       these for readability.
+            // They also might depend on build parameters, which would make
+            // using a fixed number of frames otherwise brittle.
+            version (LDC) enum BaseExceptionFunctionName = "_d_throw_exception";
+            else          enum BaseExceptionFunctionName = "_d_throwdwarf";
+            if (!startIdx && locations[i].procedure == BaseExceptionFunctionName)
+                startIdx = i;
```

Comment:
`startIdx = i + 1` to exclude `_d_throw*`?

Reply:
Fixed